core/src/main/scala/org/broadinstitute/dsde/rawls/entities/EntityService.scala

```diff
@@ -604,14 +605,18 @@ class EntityService(protected val ctx: RawlsRequestContext,
         // Else, check if the user is an owner of the workspace; retrieve it if so.
         workspaceContext <- traceFutureWithParent("getV2WorkspaceContextAndPermissions", s) { _ =>
           if (userIsAdmin) {
-            logger.info(s"Executing Quicksilver migration as admin for $workspaceName")
-            getV2WorkspaceContext(workspaceName, Some(WorkspaceAttributeSpecs(all = false)))
+            getV2WorkspaceContext(workspaceName, Some(WorkspaceAttributeSpecs(all = false))) map { ctx =>
+              logger.info(s"Quicksilver migration ${ctx.workspaceId} executing as admin for $workspaceName")
+              ctx
+            }
```

Comment:
I changed a bunch of logs so they all start with `Quicksilver migration $workspaceId`. When watching logs via cloud console Log Explorer, this should really help!